assets/app/views/_triggers.html

```diff
@@ -1,6 +1,10 @@
 <div class="builds-block">
   <div class="builds" ng-repeat="trigger in triggers">
-    <div ng-repeat="build in trigger.builds" ng-if="build.status.phase == 'New' || build.status.phase == 'Pending' || build.status.phase == 'Running' || (build.metadata.creationTimestamp | ageLessThan : 5 : 'minutes')" class="build animate-repeat osc-object" kind="Build" resource="build">
+    <div ng-repeat="build in trigger.builds"
```

Comment:
We might want to look at track by here. This loops over every build ever for the project. Hopefully people prune.